www/Gemfile.lock

```diff
@@ -1,275 +0,0 @@
-PATH
```

Comment:
Can you please revert this change from your commit? The Gemfile.lock for the `www` should remain, and if there is a desire to drop it, we should do that as a separate PR. Thanks!